diff --git a/node_modules/bootstrap/dist/js/bootstrap.bundle.js b/node_modules/bootstrap/dist/js/bootstrap.bundle.js
index 93cbd3f..835dd15 100644
--- a/node_modules/bootstrap/dist/js/bootstrap.bundle.js
+++ b/node_modules/bootstrap/dist/js/bootstrap.bundle.js
@@ -5202,8 +5202,10 @@
         if (!this._isHovered) {
           this._disposePopper();
         }
-        this._element.removeAttribute('aria-describedby');
-        EventHandler.trigger(this._element, this.constructor.eventName(EVENT_HIDDEN$2));
+        if (this._element) {
+          this._element.removeAttribute('aria-describedby');
+          EventHandler.trigger(this._element, this.constructor.eventName(EVENT_HIDDEN$2));
+        }
       };
       this._queueCallback(complete, this.tip, this._isAnimated());
     }
diff --git a/node_modules/bootstrap/dist/js/bootstrap.esm.js b/node_modules/bootstrap/dist/js/bootstrap.esm.js
index 04807b9..b6e754e 100644
--- a/node_modules/bootstrap/dist/js/bootstrap.esm.js
+++ b/node_modules/bootstrap/dist/js/bootstrap.esm.js
@@ -3361,8 +3361,10 @@ class Tooltip extends BaseComponent {
       if (!this._isHovered) {
         this._disposePopper();
       }
-      this._element.removeAttribute('aria-describedby');
-      EventHandler.trigger(this._element, this.constructor.eventName(EVENT_HIDDEN$2));
+      if (this._element) {
+        this._element.removeAttribute('aria-describedby');
+        EventHandler.trigger(this._element, this.constructor.eventName(EVENT_HIDDEN$2));
+      }
     };
     this._queueCallback(complete, this.tip, this._isAnimated());
   }
diff --git a/node_modules/bootstrap/js/dist/tooltip.js b/node_modules/bootstrap/js/dist/tooltip.js
index 033dd7b..e70fe9b 100644
--- a/node_modules/bootstrap/js/dist/tooltip.js
+++ b/node_modules/bootstrap/js/dist/tooltip.js
@@ -252,8 +252,10 @@
         if (!this._isHovered) {
           this._disposePopper();
         }
-        this._element.removeAttribute('aria-describedby');
-        EventHandler.trigger(this._element, this.constructor.eventName(EVENT_HIDDEN));
+        if (this._element) {
+          this._element.removeAttribute('aria-describedby');
+          EventHandler.trigger(this._element, this.constructor.eventName(EVENT_HIDDEN));
+        }
       };
       this._queueCallback(complete, this.tip, this._isAnimated());
     }
diff --git a/node_modules/bootstrap/js/src/tooltip.js b/node_modules/bootstrap/js/src/tooltip.js
index 1ad5615..e24e799 100644
--- a/node_modules/bootstrap/js/src/tooltip.js
+++ b/node_modules/bootstrap/js/src/tooltip.js
@@ -274,8 +274,10 @@ class Tooltip extends BaseComponent {
         this._disposePopper()
       }
 
-      this._element.removeAttribute('aria-describedby')
-      EventHandler.trigger(this._element, this.constructor.eventName(EVENT_HIDDEN))
+      if (this._element) {
+        this._element.removeAttribute('aria-describedby')
+        EventHandler.trigger(this._element, this.constructor.eventName(EVENT_HIDDEN))
+      }
     }
 
     this._queueCallback(complete, this.tip, this._isAnimated())
